pkg/fileutil/lock_linux.go

```diff
@@ -30,7 +30,7 @@ import (
 //
 // constants from /usr/include/bits/fcntl-linux.h
 const (
-	F_OFD_GETLK  = 37
+	F_OFD_GETLK  = 36
```

Comment:
Perhaps it'll be better to reuse golang.org/x/sys/unix instead, which have stuff like this available.

Reply:
agree with you.

Reply:
very well. I'm out for lack of qualification, sorry

Reply:
I suggest you pull this one to get rid of the problem, then add your cleanup on top.

Reply:
Agree. This can be merged as-is.

Reply:
Alternatively, I have created https://github.com/etcd-io/etcd/pull/12444 which I will rebase if this one will be merged earlier.